pkg/syncer/k8scloudoperator/k8scloudoperator.go

```diff
@@ -147,6 +168,13 @@ func (k8sCloudOperator *k8sCloudOperator) GetPodVMUUIDAnnotation(ctx context.Con
 		return true, nil
 	})
 	if err != nil {
+		if podName == "" {
```

Comment:
Can you remind me why we need this?
Are we saying that is this case, we failed to find the pod with required volume itself?
And in second case(L#178), we found the pod but annotation is not present? If that's the case, we should change the error message to something better like 

```
return nil, logger.LogNewErrorf(log, "Unable to find annotation %s on pod %s in namespace %s in timeout: %d. Err: %+v, vmUUIDLabel, podName, podNamespace, timeout, err)
```